Resources/Prototypes/Entities/Clothing/Head/soft.yml

```diff
@@ -99,6 +99,22 @@
   id: ClothingHeadHatQMsoftFlipped
   name: quartermaster's cap
     
+- type: entity
+  parent: ClothingHeadHeadHatBaseFlippable
+  id: ClothingHeadHatCommandsoft
+  name: command cap
+  description: "It's a baseball hat painted in Command colours."
```

Comment:
why is it in quotation marks?

Reply:
The quotation marks do not appear in game. Other descriptions in the files use quotation marks in the files as well.

Reply:
Huh… weird. I might make a github issue on that

Reply:
> why is it in quotation marks?

we use it when we have ' symbol in name or decs